### ISSUE TITLE
Error code 403 over remote event update using only local tags

### ISSUE THREAD

**Issue by minisephirot** [MISP/PyMISP#1125]
TODO: Context
TODO: Here my perfect python script

**Comment by adulau** [comment]
If you have a 403 error, it’s a perm issue and not really an issue with PyMISP.

**Comment by iglocska** [comment]
Based on the provided information, I can safely deduce it's DNS. It's always DNS.

**Comment by minisephirot** [comment]
I've completed my issue, sorry for the delay :)

**Comment by minisephirot** [comment]
It seems to be related to those issues: https://github.com/MISP/MISP/issues/4691 & https://github.com/MISP/MISP/issues/4925
My user is in host organisation, he tries via API to update an event using **only** local tags -> 403

**Comment by minisephirot** [comment]
@adulau 
> If you have a 403 error, it’s a perm issue and not really an issue with PyMISP.
I thought so too but since MISP allowed me in GUI to edit local tags I thought this was more of a MISP <-> PyMISP interaction

@Rafiot 
Does this issue still needs more information ?

**Comment by Rafiot** [comment]
@minisephirot no, it doesn't. But it is going to be a MISP API issue, I don't think there is a fix on PyMISP side. 

cc @iglocska

**Comment by minisephirot** [comment]
@Rafiot Thank you for the feedback.

Should I close this issue and reopen it on MISP's repo or let someone transfer it ?

**Comment by Rafiot** [comment]
If it works as site admin, it is definitely a perm issue.

Can you check the tag settings and make sure it is not limited to a specific user/org and you're trying to add it from another one?

**Comment by minisephirot** [comment]
What do you mean by tag settings ? 
I've tried going from:
`misp_event.add_tag(name = tag, local = True)`
to
`attribute.add_tag(name = tag, local = True, exportable = True)`
with no significant change in MISP behaviour:  is there some settings in MISP to change ?

**Comment by minisephirot** [comment]
Hello, thank you for the insight. 
I thought that calling an update with only tags beeing changed would not trigger a "true" event update like an edition of attributes : that's a good thing to know.
I specificaly avoided misp_instance.(un)tag because it directly called a request http and wanted to bulk tag/untag, I'll try using the method and see if this goes throught !
